### PR TITLE
Group wiki

### PR DESCRIPTION
**Pull request by rosecers**
@arthur-lin1027 and @saswatnayak1998 can you let me know what you think of this? Are there other pages that would be good to add?

I also added instructions for locally building the website to check it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 5. Add a professional photo (preferably square aspect ratio) at `assets/img/your_name.png`.
 6. Add any extra photos (research, hobbies, etc.) at `assets/img/your_name_#.png`., where `#` is a number 1-3. If you choose not to do this, comment out the table in `members/your_name.md`.
 7. Run `make_member_page.py` and check that there are no errors.
-8. Commit your branch and push it to the repository with `git add *; git commit -m "Making a page for your name"; git push origin your_name_branch`.
-8. Create a pull request on `https://github.com/cersonsky-lab/website/pulls` and check that all tests run.
-9. Request one of your group members and Rosy as a reviewer.    
+8. Preview your webpage by calling `bundle install; bundle exec jekyll serve --baseurl "website"`.
```
Comment (rosecers) [inline README.md:9]:
```suggestion
8. Preview your webpage by calling `bundle install; bundle exec jekyll serve --baseurl "/website"`.
```

**Review by saswatnayak1998 — APPROVED** [review]

**Comment by rosecers** [comment]
@arthur-lin1027 does this look good to you too?

**Review by arthur-lin1027 — APPROVED** [review]
Maybe the only thing to add to README.md would be to add instructions for removing DS_STORE so reviewers don't always have to comment about that? On the other hand, it's good practice for the reviewers haha.

**Comment by rosecers** [comment]
I added the `DS_Store` to the gitignore (just now), so hopefully that won't be an issue from hereon

**Pull request merged**